Framework/CurveFitting/test/Algorithms/LeBailFitTest.h

```diff
@@ -30,6 +21,642 @@ using namespace std;
+       16099.907910, 16114.821630, 16129.749160, 16144.690520, 16159.645720,
+       16174.614780, 16189.597700, 16204.594500, 16219.605190, 16234.629790,
+       16249.668300, 16264.720740, 16279.787130, 16294.867480, 16309.961790,
+       16325.070090, 16340.192380, 16355.328680, 16370.479000, 16385.643350,
+       16400.821750, 16416.014210, 16431.220750, 16446.441370, 16461.676090,
+       16476.924920, 16492.187880, 16507.464970, 16522.756220, 16538.061630,
+       16553.381220, 16568.715010, 16584.062990, 16599.425190, 16614.801630,
+       16630.192300, 16645.597230, 16661.016440, 16676.449920, 16691.897700,
+       16707.359800, 16722.836210, 16738.326960, 16753.832060, 16769.351530,
+       16784.885370, 16800.433590, 16815.996220, 16831.573270, 16847.164750,
+       16862.770670, 16878.391040, 16894.025890}};
+  std::array<double, 73> vecY = {
+      {0.000939, 0.003453, -0.000912, 0.001885, 0.003328, 0.003645, 0.002186,
+       0.001818, 0.001830, 0.002610, 0.007754, 0.011196, 0.021295, 0.034910,
+       0.069452, 0.119978, 0.213131, 0.328728, 0.463766, 0.606728, 0.709954,
+       0.727371, 0.680923, 0.561676, 0.426857, 0.302604, 0.207706, 0.146549,
+       0.096288, 0.069523, 0.044938, 0.031268, 0.024555, 0.020716, 0.014238,
+       0.010839, 0.009522, 0.006665, 0.004833, 0.006066, 0.007979, 0.003380,
+       0.006960, 0.000764, -0.001748, 0.003117, 0.002671, 0.000732, 0.001814,
+       -0.000605, -0.003475, 0.003512, -0.000796, 0.006515, 0.010276, 0.004984,
+       0.006924, 0.007722, 0.006036, 0.003330, 0.002929, 0.007368, 0.001504,
+       0.002403, 0.004263, 0.001860, 0.002712, 0.001574, -0.001803, 0.000825,
+       -0.003368, -0.003277, -0.001997}};
+  std::array<double, 73> vecE = {
+      {0.001830, 0.001826, 0.001835, 0.001824, 0.001851, 0.001836, 0.001845,
+       0.001869, 0.001882, 0.001898, 0.001915, 0.001932, 0.001969, 0.002054,
+       0.002229, 0.002469, 0.002831, 0.003231, 0.003662, 0.004061, 0.004333,
+       0.004400, 0.004303, 0.004013, 0.003638, 0.003256, 0.002927, 0.002681,
+       0.002477, 0.002343, 0.002272, 0.002194, 0.002167, 0.002138, 0.002107,
+       0.002104, 0.002092, 0.002101, 0.002102, 0.002085, 0.002110, 0.002091,
+       0.002097, 0.002122, 0.002122, 0.002117, 0.002126, 0.002175, 0.002159,
+       0.002176, 0.002180, 0.002188, 0.002204, 0.002243, 0.002229, 0.002247,
+       0.002239, 0.002232, 0.002285, 0.002255, 0.002310, 0.002281, 0.002326,
+       0.002273, 0.002314, 0.002311, 0.002316, 0.002333, 0.002348, 0.002338,
+       0.002344, 0.002340, 0.002348}};
+
+  // b) Get workspace
+  const size_t size = vecX.size();
+  auto dataws = boost::dynamic_pointer_cast<API::MatrixWorkspace>(
+      API::WorkspaceFactory::Instance().create("Workspace2D", 1, size, size));
+
+  // c) Input data
+  for (size_t i = 0; i < size; ++i) {
+    dataws->dataX(0)[i] = vecX[i];
+    dataws->dataY(0)[i] = vecY[i];
+    dataws->dataE(0)[i] = vecE[i];
+  }
```

Comment:
@raquelalvarezbanos Sorry, my bad, you need `Points` instead of `BinEdges` (as I should have seen from the workspace creation which uses `size` for both X and Y length).